webserver/Dockerfile

```diff
@@ -41,6 +41,9 @@ RUN printenv >> /etc/environment
 COPY . .
 
 # CMD ["supervisord", "-c", "/etc/supervisor.conf"]
-RUN virtualenv -p python3.7 restapi
+# RUN virtualenv -p python3.7 restapi
+
+RUN restapi/bin/pip3 install pip --upgrade
```

Comment:
please move this above copy or below RUN apt-get

Reply:
Resolved